### ISSUE TITLE
MotioneyeOS as AP using RaspberryPi3B+

### ISSUE THREAD

**Issue by BruceRU** [ccrisan/motioneyeos#1681]
close to item #1503

### Preliminary Docs

I confirm that I have read the [CONTRIBUTING] -
 yes
(https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.
yes

### motionEyeOS Version

I am running motionEyeOS version: motioneyeos-raspberrypi3-20180627.img

### Board Model

I am using the following board/model: Raspberry PI 3B+).

### Camera

No camera

My camera model is: n/a

### Network Connection

My motionEyeOS unit is connected to the network via: No connection

### Peripherals

I am using the following peripherals that I consider relevant to this issue:

 * USB keyboard
 * HDMI monitor

### Log Files

I consider the following log files relevant to this issue:
where do I find these?  I am a Noob (please forgive me).

 * (attach e.g. motioneye.log)
 * (attach e.g. boot.log)
 
The following Word doc contains my process for trying to create a MotionEyeOS access point
[MotionEyeOS Access point.docx](https://github.com/ccrisan/motioneyeos/files/2487284/MotionEyeOS.Access.point.docx)

A camera screen shot of the HDMI monitor:

![20181015_194209](https://user-images.githubusercontent.com/30234819/47085816-c6556000-d1e5-11e8-81d2-458bed6d310b.jpg)

**Comment by ccrisan** [comment]
You must connect your system to the network, either via wifi or ethernet. If you really don't want a network connection, set [os_networkless](https://github.com/ccrisan/motioneyeos/wiki/os.conf#os_networkless) to true.

**Comment by ccrisan** [comment]
Also check out this article: https://github.com/ccrisan/motioneyeos/wiki/WiFi-Access-Point.

**Comment by BruceRU** [comment]
Calin,
Thanks for doing this, it's an AWESOME achievement!
I found the source of my issue above.  Having created the 3 required boot files.  When I renamed them, they still had the TXT file type instead of the CONF file type.
After renaming and rebooting, the command "os_networkless" was not found.
See the screen capture below.

![image](https://user-images.githubusercontent.com/30234819/47154665-edc52f00-d2b0-11e8-8285-4ad7057c2b5f.png)

Thanks again.
